fe/fe-core/src/main/java/org/apache/doris/catalog/ColumnType.java

```diff
@@ -115,7 +114,8 @@ public static void write(DataOutput out, Type type) throws IOException {
         Preconditions.checkArgument(type.isScalarType(), "only support scalar type serialization");
         ScalarType scalarType = (ScalarType) type;
         if (scalarType.getPrimitiveType() == PrimitiveType.DECIMALV2) {
-            Text.writeString(out, PrimitiveType.DECIMAL.name());
+            // todo rename decimalV2 to decimal. then use PrimitiveType.DECIMALV2.name()
+            Text.writeString(out, "DECIMAL");
```

Comment:
Here we can just write DECIMALV2. And in read method, check if it is DECIMAL or DECIMALV2